src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -410,6 +410,72 @@ const char* Hole::ThreadSize_ISOmetricfine_Enums[]   = {
     "M100x2.0",    "M100x3.0",    "M100x4.0",    "M100x6.0", NULL };
 const char* Hole::ThreadClass_ISOmetricfine_Enums[]  = { "4G", "4H", "5G", "5H", "6G", "6H", "7G", "7H","8G", "8H", NULL };
 
+/* ISO 4762 Counterbore */
+const Hole::CounterDimension Hole::iso4762_data[] = {
+    { 2,   4.3,  2.1 },
+    { 2.5, 5.0,  3.0 },
+    { 3,   6.0,  3.4 },
+    { 3.5, 6.5,  3.9 },
+    { 4,   8.0,  4.4 },
+    { 5,  10.0,  5.4 },
+    { 6,  11.0,  6.4 },
+    { 8,  15.0,  8.6 },
+    { 10, 18.0, 10.6 },
+    { 12, 20.0, 12.6 },
+    { 14, 24.0, 14.6 },
+    { 16, 26.0, 16.6 },
+    { 18, 30.0, 18.6 },
+    { 20, 33.0, 20.6 },
+    { 22, 36.0, 22.8 },
+    { 24, 40.0, 24.8 },
+    { 27, 46.0, 31.0 },
+    { 30, 50.0, 34.0 },
+    { 33, 54.0, 36.0 },
+    { 36, 58.0, 37.0 },
+    {  0,  0,    0   } };

```

Comment:
sorry to be OCD but commas should line up and numbers should line up right to left
```
    { 36, 58.0, 37.0 },
    {  0,    0,    0 } };